src/components/DateForm.tsx

```diff
@@ -0,0 +1,89 @@
+import {
+  type TextInputRefType,
+  Fieldset,
+  TextInput,
+  Notification,
+} from "@nypl/design-system-react-components"
+import type { SyntheticEvent, MutableRefObject, Dispatch } from "react"
+import { debounce } from "underscore"
+
+type DateFormPropsType = {
+  inputRef?: MutableRefObject<TextInputRefType>
+  dateAfter: string
+  dateBefore: string
+  debounceInterval: number
+  setDateRangeError: Dispatch<React.SetStateAction<boolean>>
+  changeHandler: (e: SyntheticEvent) => void
+  displayDateRangeErrorNotification: boolean
+}
+
+const DateForm = ({
+  inputRef,
+  dateAfter,
+  dateBefore,
+  debounceInterval,
+  changeHandler,
+  // these two params, though they sound similar are not
```

Comment:
Why not? That's good information to have as comments here.

Reply:
updated comment